images/kibana/docker-compose.production.yml

```diff
@@ -0,0 +1,6 @@
+version: "2"
```

Comment:
Having `production` as part of the filename is misleading. If this is just for testing then why not consolidate them into a single `.test.yml`?